_sidebar.md

```diff
@@ -1,7 +1,10 @@
 - [**Introduction**](/README.md)
 - [**Usage Tips**](/usage-tips.md)
-- [**JSON RPC**](/json-rpc.md)
 - [**Plugin Directory**](/plugins.md)
-- [**Porting Plugins**](/port-plugins.md)
-- [**Develop C# Plugins**](/develop-csharp-plugins.md)
-- [**Develop Python Plugins**](/develop-py-plugins.md)
+- Plugin Development
+    - [**plugin.json**](/plugins.json.md)
+    - [**JSON RPC**](/json-rpc.md)
+        - [**Develop Python Plugins**](/develop-py-plugins.md)
```

Comment:
click https://zeroto521.github.io/docs/#/json-rpc and https://zeroto521.github.io/docs/#/develop-py-plugins.

Compare to the sidebar changes.

You would see what I have said at up.

Reply:
Got you